docs: added the 2023 mentorship idea list and updated the status of the mentorship program. (#759)
asyncapi/community: mentorship/asyncapi-mentorship/2023/README.md

# Status: In Progress

## Timeline

- Project proposals applications open: April 1st - June 6th

> - Mentees applications open: June 6th - June 23th

Reviewer: ```suggestion
- Mentees applications open: June 6th - June 23rd
```

> - Application review by the mentors/admission decisions: June 23th - June 28th

Reviewer: ```suggestion
- Application review by the mentors/admission decisions: June 23rd - July 7th
```

Mentorship duration - Six months \(full-time schedule\)

## Accepted projects
| No | Project Idea | Mentor/s | Category | Mentee |
| --- | --- | --- | --- | --- |
| 1 | [Add help/{command} endpoint](https://github.com/asyncapi/server-api/issues/144) | [David Pereira](https://github.com/BOLT04) | Engineering | [@princerajpoot20](https://github.com/princerajpoot20) |
| 2 | [Website UI Kit design/dev project](https://github.com/asyncapi/design-system/issues/4) | [AceTheCreator](https://github.com/AceTheCreator) | Design | [@Mayaleeeee](https://github.com/Mayaleeeee) |
| 3 | [MVP integration of extensions catalog with AsyncAPI tools](https://github.com/asyncapi/extensions-catalog/issues/78) | [Lukasz Gornicki](https://github.com/derberg) | Engineering | [@sambhavgupta0705](https://github.com/sambhavgupta0705) |

> | 4 | [Documenting how different protocols work with AsyncAPI](https://github.com/orgs/asyncapi/discussions/533) | [Alejandra Quetzalli](https://github.com/alequetzalli) | Documentation | [@alequetzalli](https://github.com/alequetzalli) |

Reviewer: ```suggestion
| 4 | [Documenting how different protocols work with AsyncAPI](https://github.com/orgs/asyncapi/discussions/533) | [Alejandra Quetzalli](https://github.com/alequetzalli) | Documentation |  To be announced (TBA) |
```
